api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -347,9 +360,14 @@ void shouldHandleNullDeviceFingerprint() {
         var entitlement = new PaymentCycleVoucherEntitlement(asList(firstVoucherEntitlement, secondVoucherEntitlement));
         EligibilityAndEntitlementDecision decision = aDecisionWithStatusAndEntitlement(ELIGIBLE, entitlement);
         given(eligibilityAndEntitlementService.evaluateClaimant(any())).willReturn(decision);
+        ClaimRequest claimRequest = ClaimRequest.builder()
```

Comment:
Looks like you could use a Test Data Factory here and for the two tests below too.

Reply:
done